tests/vm_startup_test.go

```diff
@@ -0,0 +1,162 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+const timeout = 3 * time.Minute
+const pollingInterval = 5 * time.Second
+const testNamespace = "secondary-test"
+const dnsPort = "31111"
+const dnsIP = "127.0.0.1" // Forwarded to the node port - https://github.com/kubevirt/kubevirtci/pull/867 ,https://github.com/kubevirt/kubesecondarydns/pull/23
```

Comment:
There is no chance that external contributor or even future us will understand why "127.0.0.1" is the dnsIP without thos comment.

Reply:
so maybe just something generic as this is the flow of kubevirtci  ?
or even just the 2nd PR can be enough ?
two PRs is way too much imo

Reply:
I think those two PR explain the best how we rich the dns. I prefer to have it and to same future developer some time in the future.

Reply:
I am personally against too much comments / links
if we can find something in between it would be great please

Reply:
note that even now the PRs are outdated, because the was another PR which change kuevirtci default host port from 53 to 31111
this is the problem with comments

I think we can mention here the README.md, that once the PR of readme is merged, it explains a bit about the topology we are using on kubevirtci